src/main/java/com/minecolonies/achievements/AchBuildingFisher.java

```diff
@@ -0,0 +1,18 @@
+package com.minecolonies.achievements;
+
+import com.minecolonies.blocks.ModBlocks;
+import net.minecraft.block.Block;
+import net.minecraft.stats.Achievement;
+
+/**
+ *
+ * @author Isfirs
+ * @since 0.2
+ */
+public class AchBuildingFisher extends AbstractAchievement
+{
+    public AchBuildingFisher(final String id, final String name, final int offsetX, final int offsetY)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)